app/javascript/shared/transfersLayouts.spec.js

```diff
@@ -1,16 +1,16 @@
-import { transfersFromRequests } from './transfersLayouts'
+import { transfersFromRequests, buildPlatesMatrix, buildLibrarySplitPlatesMatrix, buildSequentialLibrarySplitTransfersArray } from './transfersLayouts'
```

Comment:
I can't help but think that you may be better off keeping some of these methods private, and not testing them. Instead just test the exposed public behaviours. It may feel like this is reducing test coverage, but ultimately it improve the ability to make internal changes, as we're only testing the public interface.

Reply:
In this method we tried creating the code and tests at the same time to help in the development. The problem is this methods at the end are very specific. Mainly the tests were for helping the development, but still there is some value in keeping the tests for this and can be removed if not needed.